src/status_im/ui/components/chat_icon/styles.cljs

```diff
@@ -64,6 +64,12 @@
           :height 40
           :margin 0}))
 
+(defn custom-size-icon [size]
+  (merge chat-icon
+         {:width  size
+          :height size
+          :margin 0}))
```

Comment:
Do we want this margin for all icons?

Reply:
we don't want `:margin        4` for `custom-size-icon `

Reply:
Do we want it for all other icons in general? Maybe it's just a specific types of icons?

Reply:
this is a specific icon for contacts , i would say it's not an icon at all, it has a little icon in the corner, it shows contact's status - online, dapp, etc, and all these "icons" have margin , but for my "icons" i don't need this margin, we should review this ns separately and create universal contact badge component

Reply:
Makes sense. Maybe worth an issue to keep track of it?

Reply:
this ns is mostly used in chat @goranjovic jfy if you will be working with it

Reply:
https://github.com/status-im/status-react/issues/3205